modules/oe_theme_helper/tests/src/Functional/NodeViewRoutesMetadataTest.php

```diff
@@ -88,16 +88,16 @@ public function testNodeRoutes(): void {
     // Verify that the page header block is shown in the node canonical route
     // and contains the correct revision text.
     $this->drupalGet($node->toUrl());
-    $this->assertSession()->elementTextContains('css', '.ecl-page-header__intro', $published_revision_body);
+    $this->assertSession()->elementTextContains('css', '.ecl-page-header__description', $published_revision_body);
 
     // Verify that the block is also shown in the latest version route with the
     // correct draft revision loaded.
     $this->drupalGet($node->toUrl('latest-version'));
-    $this->assertSession()->elementTextContains('css', '.ecl-page-header__intro', $draft_revision_body);
+    $this->assertSession()->elementTextContains('css', '.ecl-page-header__description', $draft_revision_body);
 
     // Verify also for the node single revision route.
     $this->drupalGet($first_revision_url);
-    $this->assertSession()->elementTextContains('css', '.ecl-page-header__intro', $published_revision_body);
+    $this->assertSession()->elementTextContains('css', '.ecl-page-header__description', $published_revision_body);
```

Comment:
Out of scope but we can keep them, since 2.x is such a moving target.

Reply:
:+1: thanks
The purpose is to add a maximum of PHPUnit tests.